src/openforms/js/components/admin/form_design/registrations/objectsapi/LegacyConfigFields.js

```diff
@@ -183,7 +183,10 @@ const LegacyConfigFields = ({index, name, schema, formData, onFieldChange}) => {
           id="root_uploadSubmissionCsv"
           name="uploadSubmissionCsv"
           value={uploadSubmissionCsv}
-          onChange={onFieldChange}
+          checked={uploadSubmissionCsv}
```

Comment:
I haven't checked but can't this `checked` prop be inferred by the `Checkbox` component itself from the `value` prop?

Reply:
If I remove it from here it doesn't appear in the html. And in the component itself (https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/components/admin/forms/Inputs.js#L99) I don't see `value` (it comes from the extra props) so how can this be inferred?Could you please elaborate?

Reply:
`value` is pretty meaningless for checkboxes, did a checkup on the HTML spec. In non-react code, the `value` is what would be sent to the server when the checkbox is checked, but in React this doesn't make any sense - we just track whether it's checked or not, so probably best to omit `value` entirely for the checkbox. All that matters is `checked`.